source/v4/users.yaml

```diff
@@ -581,3 +581,48 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  /users/{user_id}/mfa:
+    put:
+      tags:
+        - users
+      summary: Update a user's mfa
+      description: |
+        Activates multi-factor authenticates for the current user if activate
```

Comment:
Can this go from `Activates multi-factor authenticates for the current user` to `Activates multi-factor authentication for the user`?